data/korath/korath.txt

```diff
@@ -279,15 +279,18 @@ fleet "Korath Home"
 	cargo 3
 	personality
 		timid opportunistic
-	variant 3
+	variant 90
```

Comment:
I'm no numbers person but those are some extreme odds right there.

```suggestion
	variant 30
```

Reply:
The frequency for this fleet to spawn a lot more frequent. The numbers present make sure that the 577 in its new fleet doesn’t spawn more often. Keep its original rareness.